### PR TITLE
update edit in place

### PR DESCRIPTION
**Pull request by iFun**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -976,9 +976,9 @@
       }
     },
     "@brightspace-ui-labs/edit-in-place": {
-      "version": "1.0.10",
-      "resolved": "https://registry.npmjs.org/@brightspace-ui-labs/edit-in-place/-/edit-in-place-1.0.10.tgz",
-      "integrity": "sha512-uV0MoPs2XkMKYX3kSgEieUZBaTqV4iodN6P3U+IlXLxNPlhm/EulodqP+nQ3jbTpA7/PuT2Ka0AYBM4L8MWYbQ==",
+      "version": "1.0.11",
```
Comment (AKobets) [inline package-lock.json]: is this the only change needed? how about polymer.json?

Comment (iFun) [reply package-lock.json]: I think this is the only place needs to be changed

Comment (BhamelinD2L) [reply package-lock.json]: Activities dependency of edit in place is not updated either. If anyone else is using edit in place, their dependency is out of date as well. Please review the onboarding notes on how to update BSI correctly : https://desire2learn.atlassian.net/wiki/spaces/SPACE/pages/925076688/Updating+the+BSI

You also needed to edit the extra dependencies in polymer.json  as described by the Rally task TA140590, unless that's another PR.

Comment (AKobets) [reply package-lock.json]: lets make all changes to bsi in 1 PR

**Comment by iFun** [comment]
will update PR when https://github.com/BrightspaceHypermediaComponents/activities/pull/706 is merged

File: package-lock.json [hunk 2]
```diff
@@ -4272,16 +4261,12 @@
       }
     },
     "d2l-menu": {
-      "version": "github:BrightspaceUI/menu#0f12702460b33981a0285f24e772c7f440ee9f6d",
+      "version": "github:BrightspaceUI/menu#437d7efdca44a3b5f5e1611acd2103df72ed47ef",
```
Comment (iFun) [inline package-lock.json]: @margaree

File: package-lock.json [hunk 3]
```diff
@@ -4294,7 +4279,7 @@
       }
     },
     "d2l-my-courses": {
-      "version": "github:Brightspace/d2l-my-courses-ui#26108f11d254b10f07e3625d57a890d60aa03999",
+      "version": "github:Brightspace/d2l-my-courses-ui#61cf26d72f874c582f199a47cf38ce1c3891a532",
```
Comment (iFun) [inline package-lock.json]: @AllanKerr

File: package-lock.json [hunk 4]
```diff
@@ -4631,7 +4616,7 @@
       }
     },
     "d2l-sequences": {
-      "version": "github:BrightspaceHypermediaComponents/sequences#cf455c740e9dc229049c52bb9f2b5647d1d3e347",
+      "version": "github:BrightspaceHypermediaComponents/sequences#7e893c7cade327ab5f4ccb6ac86d8472c2c386fc",
```
Comment (iFun) [inline package-lock.json]: @ChrisKraljevicD2L

File: package-lock.json [hunk 5]
```diff
@@ -4056,13 +4056,12 @@
       }
     },
     "d2l-dropdown": {
-      "version": "github:BrightspaceUI/dropdown#59c7a64540db0d2587baf24d944b696e5e379f4c",
+      "version": "github:BrightspaceUI/dropdown#482055e65e849e05ee779c3beba56ec9bf76c839",
```
Comment (iFun) [inline package-lock.json]: @AllanKerr

**Comment by iFun** [comment]
Another PR updated the packages so I just end up updating the polymer.json file

**Pull request merged**